M2/Macaulay2/tests/normal/formatting.m2

```diff
@@ -8,3 +8,9 @@ assert (
 QQ[a][x]
 assert( toString (x-a) == "x-a" )
 assert( class expression (-a) === Minus )
+
+-- html formatting
+debug Core
```

Comment:
Why is this line needed?

Reply:
TEX is not exported by default, and `debug Core` is faster than `needsPackage "Text"`.